### ISSUE TITLE
No adapter detected - WP8.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [pybluez/pybluez#72]
```
What steps will reproduce the problem?
1. Use code from Inquiry.py example
2. returns line 89, in handleBroadcast
    nearby_devices = bluetooth.discover_devices(lookup_names = True)
  File "C:\Python27\lib\site-packages\bluetooth\msbt.py", line 9, in discover_devices
    return bt.discover_devices (flush_cache, lookup_names, lookup_class)
IOError: No Bluetooth adapter detected


What is the expected output? What do you see instead?
The expected output is that the bluetooth adapter is found. It is included on 
the Microsoft bluetooth stack (At hardware>devices it says Microsoft Bluetooth 
Enumerator)

What version of the product are you using? On what operating system?
PyBluez-0.20 (win32) for Python 2.7
Python 2.7
Windows 8.1

Please provide any additional information below.
No clue whatsoever why the adapter isn't visible for pyBluez.

```

Original issue reported on code.google.com by `Solouwma...@gmail.com` on 1 Oct 2014 at 12:11

**Comment by stcalica** [comment]
This may be related. I am using Windows 8.1, Python 2.7 on a Toshiba Thinkpad. 
I installed Pybluez as a wheel through pip, and after installation it worked great and all the examples worked. 

Now out of nowhere I get that same response "No Bluetooth adapter detected." And I even tried the gui and that did not work.

**Comment by karulis** [comment]
It looks like this happens when there are no BT devices to discover

And here is the answer:
http://stackoverflow.com/questions/19762847/windows-bluetooth-wsalookupservicebegin-returns-wsaservice-not-found-10108-if

**Comment by karulis** [comment]
Fixed by commit d89eddac3f0d3b1400b17f5e09af98af40ba0a61
